### ISSUE TITLE
Upload genome files to the website

### ISSUE THREAD

**Issue by hushaoqiang** [aaranyue/quarTeT#3]
Hello, I am very grateful to see that you have developed this software. At present, I want to use the website to identify the Centromere sequence. But when I upload the genome file, it always disappears, and when I run it, I prompt that I haven't entered the reference genome file.

How should this be resolved? Looking forward to your reply. Thank you.

![2](https://github.com/aaranyue/quarTeT/assets/68465643/a0db7575-f639-44a1-97f0-a10c5dad10a9)
![3](https://github.com/aaranyue/quarTeT/assets/68465643/e0c0d963-fc15-4c93-be3b-37af6be8b711)
![4](https://github.com/aaranyue/quarTeT/assets/68465643/a024121e-4d46-4d2e-b3f1-9ece03596559)

**Comment by Echoring** [comment]
Sorry to say, the storage of our server is unexpectedly filled...
We have cleared up the server, please try again.

**Comment by hushaoqiang** [comment]
Hello, there are still some errors here.
![1](https://github.com/aaranyue/quarTeT/assets/68465643/7c9d8517-046c-44b8-b9b4-f77d802c3155)

**Comment by Echoring** [comment]
Sorry again to say, this is an unsolved bug. Uploaded file rarely become `null` in this bug, and we haven't located the true error. Fortunately, this bug happen rarely. If you see error such as `No such file or directory: '../../upload/null'`, just try again.
(Seems taking a long time in the submitting page or upload large file will increase happen rate)

**Comment by hushaoqiang** [comment]
Indeed, I re-entered it and now there are no errors reported.
Thanks

**Comment by hushaoqiang** [comment]
Hello, my genome has successfully run, but opening it has no results. May I know how to solve this problem
![1](https://github.com/aaranyue/quarTeT/assets/68465643/adfa48b3-52d0-441a-83e6-9728d21e9cc6)
![2](https://github.com/aaranyue/quarTeT/assets/68465643/5852264a-13cc-4b2f-b9b0-a0ab250f1f2c)

**Comment by Echoring** [comment]
However, I can successfully access your result with Job ID: Centro_72d43c2a-e2cc-4a04-beec-fa0aa6a7f6df.
You may check is there any input error, or force refresh page (ctrl+F5) and try again.

**Comment by Echoring** [comment]
* I find the error. The link in mail is outdated address. Use this one: http://atcgn.com:8080/quarTeT/home.html

**Comment by hushaoqiang** [comment]
According to what you said, I successfully opened it. The result was very good, thank you very much. I would like to ask how long this result will be saved.

**Comment by Echoring** [comment]
The result will be removed after 30 days. However, depends on server storage, the time is not guaranteed.
It's recommended to save the results by clicking the download buttoms.

**Comment by hushaoqiang** [comment]
Hello, thank you very much for providing this software（quarTeT） for scientific researchers. Wishing you better and better

**Comment by hushaoqiang** [comment]
Hello, I saw that you fixed a bug today. When I was running the webpage today, there were some question marks (TElength	TEcoverage is ?%) in the results. Is this normal?
![1](https://github.com/aaranyue/quarTeT/assets/68465643/96bf759a-a9cb-4e03-a0e8-82f43fca9ba7)

**Comment by Echoring** [comment]
If you haven't uploaded TE files, there will be question marks at TElength and TEcoverage.
The bug mentioned is imported to server at 7/4 noon. All jobs submitted after that time have failed, so I think this is not related.

**Comment by hushaoqiang** [comment]
Indeed, when I added the annotation file for EDTA, the result appeared. thanks

**Comment by Echoring** [comment]
It seems that our server is overwhelming by complex centromere identification tasks and the memory run out.
Please try the local version, or try again days latter.

**Comment by hushaoqiang** [comment]
Okay, I'll try again in a few days. Thanks
